### PR TITLE
chore: allow test to pass even if commit name is not good

### PR DESCRIPTION
**Pull request by ZiniuYu**
Follows https://github.com/jina-ai/jina/pull/5371

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -8,46 +8,10 @@ jobs:
...
-          body: |
-            Thanks for your contribution :heart:
-            :broken_heart: Unfortunately, this PR has one ore more **bad commit messages**, it can not be merged. To fix this problem, please refer to:
-            - [Commit Message Guideline for the First Time Contributor](https://github.com/jina-ai/jina/issues/553)
-            - [Contributing Guideline](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md)
-
-            Note, other CI tests will *not* *start* until the commit messages get fixed.
-
-            This message will be deleted automatically when the commit messages get fixed.
-          reaction-type: "eyes"
```
Comment (numb3r3) [inline .github/workflows/ci.yml:50]: Is this the expected behavior to drop the warning comments? maybe it is good enough to revise the comment content

**Review by numb3r3 — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ZiniuYu) [reply .github/workflows/ci.yml:50]: I think it's ok to drop it, since we do `squash and merge`. We only need to make sure the title to meet our criteria.

Comment (ZiniuYu) [reply .github/workflows/ci.yml:50]: okk

**Review by numb3r3 — APPROVED** [review]
LGTM

**Pull request merged**